src/test/scala/edu/berkeley/ce/rockslicing/InputProcessingSpec.scala

```diff
@@ -5,169 +5,267 @@ import org.scalatest.FunSuite
 import scala.io.Source
 
 class InputProcessingSpec extends FunSuite {
+
+  def compareJointSets(jointSet1: JointSet, jointSet2: JointSet): Boolean = {
+    if (jointSet1.strike == jointSet2.strike &&
+      jointSet1.dip == jointSet2.dip &&
+      jointSet1.jointSpacing == jointSet2.jointSpacing &&
+      jointSet1.persistence == jointSet2.persistence &&
+      jointSet1.stochasticFlag == jointSet2.stochasticFlag &&
+      jointSet1.strikeStDev == jointSet2.strikeStDev &&
+      jointSet1.dipStDev == jointSet2.dipStDev &&
+      jointSet1.jointSpacingStDev == jointSet2.jointSpacingStDev &&
+      jointSet1.persistenceStDev == jointSet2.persistenceStDev
+    ) {
+      true
+    } else {
+      false
+    }
+  }
+
   test("Input file without global origin specified should produce error") {
-    val inputStr = "-1.0 0.0 0.0 0.0 0.0 0.0\n" +
-      "1.0 0.0 0.0 2.0 0.0 0.0\n" +
-      "0.0 -1.0 0.0 0.0 0.0 0.0\n" +
-      "0.0 1.0 0.0 2.0 0.0 0.0\n" +
-      "0.0 0.0 -1.0 0.0 0.0 0.0\n" +
-      "0.0 0.0 1.0 2.0 0.0 0.0\n" +
-      "1.0 0.0 0.0 0.0 0.0 0.0 1.0 0.0 0.0 0.0 0.0\n"
+    val inputStr = "0.0 0.0 0.0 2.0 2.0 2.0\n" +
```

Comment:
If you want to, you could consider creating some actual files under `src/test/resources` and then reading from them in these test cases. It might make things a bit easier to understand and maintain.

I think we have some tests somewhere that take this approach.